runtime/polkadot/src/lib.rs

```diff
@@ -2068,7 +2068,7 @@ mod test_fees {
 		// much it is going to cost.
 		use pallet_elections_phragmen::WeightInfo;
 
-		// Loser candidate lose a lot of money; sybil attack by candidates is even more expensive,
+		// Loser candidate lose a lot of money; sibyl attack by candidates is even more expensive,
```

Comment:
https://www.sciencedirect.com/topics/computer-science/sybil-attack

Reply:
I put too much trust in the `en_US.dic` :facepalm: